openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -562,5 +562,12 @@ jQuery(function () {
         import(/* webpackChunkName: "fulltext-search-suggestion" */ './fulltext-search-suggestion')
             .then(module => module.initFulltextSearchSuggestion(fulltextSearchSuggestion))
     }
+
+    // Go back redirect:
+    const goback = document.querySelector('#go-back')
+    if (goback) {
+        import (/* webpackChunkName: "delete-redirect" */ './delete-redirect')
+            .then(module => module.initGoBackRedirect(goback))
```

Comment:
```suggestion
    const backLinks = document.querySelectorAll('.go-back-link')
    if (backLinks.length) {
        import (/* webpackChunkName: "go-back-links" */ './go-back-links')
            .then(module => module.initGoBackLinks(backLinks))
```
A couple of changes here:
1. Select all elements on the page that have the `go-back-link` class, then pass them to the `init` method (which should be updated to handle a [`NodeList`](https://developer.mozilla.org/en-US/docs/Web/API/NodeList) of elements).
2. Update the name of the `init` function, and the `delete-redirect.js` file name.  When I see the name `delete-redirect.js`, it's not clear that the code is related to the "Go Back" links.  I expect that naming this something more obvious will make the code easier to maintain (especially as my memory of this PR fades over time).